### PR TITLE
fix-1109 : Make the bug_classifier script automatically download the required DB

### PR DESCRIPTION
**Pull request by jeffin143**
Handling : https://github.com/mozilla/bugbug/issues/1109

### CODE REVIEW

File: scripts/bug_classifier.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ def classify_bugs(model_name, classifier, bug_id):
         bugs = bugzilla.get(bug_id).values()
         assert bugs, f"A bug with a bug id of {bug_id} was not found"
     else:
+        bugzilla.download_bugs_db()
```
Comment (marco-c) [inline scripts/bug_classifier.py]: You can just call `db.download(bugzilla.BUGS_DB)` like the other scripts, also make sure you assert on its result

**Comment by marco-c** [comment]
Make sure you fix the linting issues too, you can use pre-commit for that.

File: scripts/bug_classifier.py (hunk 1 above)
Comment (jeffin143) [reply scripts/bug_classifier.py]: Resolved

**Comment by jeffin143** [comment]
> Make sure you fix the linting issues too, you can use pre-commit for that (see https://github.com/mozilla/bugbug#auto-formatting).

Thanks, I went through it

**Pull request merged**